NOTICE.md

```diff
@@ -1,6 +1,6 @@
 <!-- @license CC0-1.0 -->
 
-# Terms of Use
+# Copyright Notice
 
 ## Branding, logo and authors’ rights
 
```

Comment:
I can't comment on the specific line, but: if we change our use of 'font' to 'typeface', we should change it in this doc as well, no?

Reply:
Yes, we should.